discourse-doctor.rb

```diff
@@ -1,18 +1,26 @@
 #!/usr/bin/env ruby
+# coding: utf-8
 
 Dir.chdir("/") do
   require 'bundler/inline'
   require 'net/http'
   require 'net/smtp'
+  require 'pathname'
 
   gemfile(true, ui: false) do
     source 'https://rubygems.org'
 
     gem 'dnsruby'
     gem 'paint'
+    gem 'sys-filesystem' # for checking available disk space
```

Comment:
When assuming that `df` and the environment (docker container) is always the same, the blocksize defaults to 1024. So no need for the `-B` option.

> Display values are in units of the first available SIZE from --block-size, and the DF_BLOCK_SIZE, BLOCK_SIZE and BLOCKSIZE environment variables.  Otherwise, units default to 1024 bytes (or 512 if POSIXLY_CORRECT is set).